dev_docs/integrations.md

```diff
@@ -0,0 +1,73 @@
+# Integrations Testing
+This documents covers the steps required to make changes to the APM package in Elastic Integrations and validate using a local Elastic Stack and a local Elastic Agent.
+
+## Use Cases
+1. Expose new APM-Server configuration to the APM Integration 
+
+## Update Integrations
+1. Follow the steps [here](https://github.com/elastic/integrations/blob/main/CONTRIBUTING.md) to fork and clone the integrations repo and make your changes 
+2. Use [elastic-package](https://github.com/elastic/elastic-package) to spin up a local stack. See integrations [quick start](https://www.elastic.co/docs/extend/integrations/quick-start) guide for more details
+- Run the below command. This will create a local stack (Kibana, ElasticSearch, Fleet Server, etc). This will also output a couple links to the local Kibana and ElasticSearch instances. It will also contain credentials to log in to the local Kibana.
```

Comment:
```suggestion
- Run the below command. This will create a local stack (Kibana, Elasticsearch, Fleet Server, etc). This will also output a couple links to the local Kibana and Elasticsearch instances. It will also contain credentials to log in to the local Kibana.
```
nit